homeassistant/components/lawn_mower/const.py

```diff
@@ -0,0 +1,36 @@
+"""Constants for the lawn mower integration."""
+from enum import IntFlag, StrEnum
+
+
+class LawnMowerActivity(StrEnum):
+    """Activity state of lawn mower devices."""
+
+    # Device is in error state, needs assistance
+    ERROR = "error"
+
+    # Paused during activity
+    PAUSED = "paused"
+
+    # Device is mowing
+    MOWING = "mowing"
+
+    # Device is in process of going back to dock
+    DOCKING = "docking"
```

Comment:
We didn't discuss the `docking` activity. I think it's good to add, but let's just check how many brands support this in the architecture issue first. I think Husqvarna supports it.